CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 3.4.0(unreleased)
+### Notes
+- This release adds support to Oneview Rest API version 800, 1000, 1200, and 1600 for C7000 and Synergy to the following features
```

Comment:
we are extending support to 1600 API.right? or we are going to add support for other resources for above API's which were not covered in previous releases?

Reply:
done